### PR TITLE
New Tiles (2 maintenance specific and 1 new wood).

### PR DESCRIPTION
**Pull request by brainfood1183**
3 new maintenance specific tiles. These tiles are designed to be used in maintenance areas to add a bit more variety to them. Whilst designed for maintenance its up to the mappers how to best utilize them.

The new rusty_maint tile is my own interpretation of the original dirty_steel tile which was great for maintenance areas, the new replacement tile is a great tile but is more subtle and suited to corridors being a slight variation of the current steel tile. This new rusty tile has a different purpose than the current dirty_steel.

Untitled-83

Untitled-94

Untitled-95

1 new wood tile. This tile is a wood pattern tiles which is similar to the current wood tile but in a distinct pattern which adds a little style to wooden floored rooms. Perfect for courtrooms or offices where you want a little variation, also works well with the new wooden border decals.

### CODE REVIEW

**Review by moonheart08 — DISMISSED** [review]
ngl the techmaint looks terrible, as stated before. Please come up with a less jarring sprite for it.
Dirty steel tiles are also too dirty.
Wood pattern is good though.

**Comment by brainfood1183** [comment]
hopefully adressed the issues you had with this pr.

**Comment by brainfood1183** [comment]
> ngl the techmaint looks terrible, as stated before. Please come up with a less jarring sprite for it. Dirty steel tiles are also too dirty. Wood pattern is good though.

updated the pr

**Comment by brainfood1183** [comment]
> Fix the validation error (just provide a link to this PR as the source)

should be fixed

**Comment by brainfood1183** [comment]
> Fix the validation error (just provide a link to this PR as the source)

anything left to do to finish this up now?

File: Resources/Textures/Tiles/attributions.yml [hunk 1]
```diff
@@ -50,3 +50,9 @@
   license: "CC-BY-SA-3.0"
   copyright: "Fortuna commit 2a9408a47e2f83d945335e4feeeeafb552173e6f, grasslight and dirt by Peptide based on grassdark.png and dirt."
   source: "https://github.com/FortunaSS13/Fortuna"
+  
+- files: ["steel_maint.png, grating_maint.png and wood_tile.png"]
```
Comment (moonheart08) [inline Resources/Textures/Tiles/attributions.yml:54]:
```suggestion
- files: ["steel_maint.png", "grating_maint.png", "wood_tile.png"]
```

**Pull request merged**